README.md

```diff
@@ -0,0 +1,161 @@
+Triage
+======
+
+Risk modeling and prediction for public policy
+
+[![image](https://travis-ci.org/dssg/triage.svg?branch=master)](https://travis-ci.org/dssg/triage)
+[![image](https://codecov.io/gh/dssg/triage/branch/master/graph/badge.svg)](https://codecov.io/gh/dssg/triage)
+[![image](https://codeclimate.com/github/dssg/triage.png)](https://codeclimate.com/github/dssg/triage)
+
+Predictive analytics projects require the coordination of many different
+tasks, such as feature generation, classifier training, evaluation, and
+list generation. These tasks are complicated in their own right, but in
+addition have to be combined in different ways throughout the course of
+the project.
```

Comment:
I feel like somewhere here we should mention the focus on problems with a time component. I don't think Triage would work without any sort of temporal config, at least as it's currently built, would it?